### PR TITLE
Updating Docker Compose 3.9 data

### PR DESCRIPTION
**Pull request by d3adwolf**
Updated the links, information and code for the current Docker Compose version 3.9.

### CODE REVIEW

**Comment by denverdino** [comment]
Thanks for your contribution

**Review by adeibnugandiSH — COMMENTED** [review]
<body> class"" pace-done light

**Review by adeibnugandiSH — COMMENTED** [review]
- _[]()_

Florian Dieminger

File: README.md [hunk 1]
```diff
@@ -3,117 +3,123 @@
...
+```yml
+version: '3.9'
 
-```
+services:
 master:
-  image: redis:3
-slave:
-  image: redis:3
-  command: redis-server --slaveof redis-master 6379
```
Comment (adeibnugandiSH) [inline README.md:27]: 1e2b5f93084694e03774554d555900ddaccf088a

**Review by adeibnugandiSH — COMMENTED** [review]
const query = @d3adwolf document.getElementById("query");
const article =@shuangkun
 document.querySelector("article");

// Find all text nodes in the article. We'll search within
// these text nodes.
const treeWalker = document.createTreeWalker(article, NodeFilter.SHOW_TEXT);
const allTextNodes = [];
let currentNode = treeWalker.nextNode();
while (currentNode) {
  allTextNodes.push(currentNode);
  currentNode = treeWalker.nextNode();
}

// Listen to the input event to run the search.
query.addEventListener("input", () => {
  // If the CSS Custom Highlight API is not supported,
  // display a message and bail-out.
  if (!CSS.highlights) {
    article.textContent = "CSS Custom Highlight API not supported.";
    return;
  }

  // Clear the HighlightRegistry to remove the
  // previous search results.
  CSS.highlights.clear();

  // Clean-up the search query and bail-out if
  // if it's empty.
  const str = query.value.trim().toLowerCase();
  if (!str) {
    return;
  }

  // Iterate over all text nodes and find matches.
  const ranges = allTextNodes
    .map((el) => {
      return { el, text: el.textContent.toLowerCase() };
    })
    .map(({ text, el }) => {
      const indices = [];
      let startPos = 0;
      while (startPos < text.length) {
        const index = text.indexOf(str, startPos);
        if (index === -1) break;
        indices.push(index);
        startPos = index + str.length;
      }

      // Create a range object for each instance of
      // str we found in the text node.

File: docker-compose.yml [hunk 2]
```diff
@@ -1,15 +1,25 @@
...
+
+services:
 master:
-  image: redis:3
+image: redis:latest
+container_name: redis-master
+
 slave:
-  image: redis:3
-  command: redis-server --slaveof redis-master 6379
```
Comment (adeibnugandiSH) [inline docker-compose.yml:5]: <body> class"" pace-done light

**Review by adeibnugandiSH — COMMENTED** [review]
ARG BUILD_FROM=alpine:3.7

FROM $BUILD_FROM

LABEL maintainer="Ian Denhardt (@d3adwolf 
(@tya 
@tealtail 
@adeibnugandiSH 

ENV PATH=/usr/local/bin:$PATH

COPY . /simp_le/src

RUN apk update \
    && apk add \
        openssl \
        python3 \
    && apk add

 --virtual .build-deps \
        gcc \
        git \
        libffi-dev \
        musl-dev \
        openssl-dev \
        py-pip \
        python3-dev \
        tzdata \

    && cp /usr/share/zoneinfo/UTC /etc/localtime \

    && echo "UTC" > /etc/timezone \

    && pip3 install --no-cache-dir /simp_le/src \

    && rm -rf /simp_le/src \
    && apk del --purge .build-deps \
    && rm -fv /var/cache/apk/*

WORKDIR /simp_le/certs

COPY /docker/entrypoint.sh /simp_le/entrypoint.sh

ENTRYPOINT [ "/simp_le/entrypoint.sh" ]
CMD [ "--help" ]

-+) sekian dokumentasi terlanjur di kerjakan berdasarkan program perangkat : 


Echo

--------------------------------------

**Pull request merged**